sideboard/configspec.ini

```diff
@@ -78,8 +78,40 @@ sqlite_dir = string(default="%(root)s/db")
 
 
 [cherrypy]
+checker.check_skipped_app_config = boolean(default=False)
+
 engine.autoreload.on = boolean(default=False)
 
+# Enable or disable the Sideboard profiler. If the profiler is enabled,
+# profiler data can be collected using the @sideboard.lib.profile decorator.
+# A web based interface will be exposed on http://servername/profiler/ to view
+# the profiler results.
+# If the profiler is disabled, then the @sideboard.lib.profile decorator
+# becomes a no-op and no performance penalty is incurred. The web interface
+# will not be created and visits to http://servername/profiler/ will 404.
+# True to enable, False to disable.
```

Comment:
Yes, the only valid values are True and False, but what do they **do**? The config setting is "profiling.on", and while it would seem obvious that "on" means "enabled" I decided to be explicit. Especially because the rest of the description uses the terms "enable" and "disable".

Take the first sentence:
> Enable or disable the Sideboard profiler.

And the last sentence:
> True to enable, False to disable.

No ambiguity there.